packages/firebase_data_connect/firebase_data_connect/lib/src/dataconnect_version.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// version number for the package, should be align with pubspec.yaml.
+const packageVersion = '0.1.0';
```

Comment:
We can probably do a script to update this version automatically on release (not important for first version)